Devscord.DiscordFramework/ControllersService.cs

```diff
@@ -121,28 +121,18 @@ private async Task RunMethodsIBotCommand(DiscordRequest request, Contexts contex
                         var commandInParameterType = method.GetParameters().First(x => typeof(IBotCommand).IsAssignableFrom(x.ParameterType)).ParameterType;
                         //TODO zoptymalizować, spokojnie można to pobierać wcześniej i używać raz, zamiast wszystko obliczać przy każdym odpaleniu
                         var template = this._botCommandsService.GetCommandTemplate(commandInParameterType);
-                        var isToContinue = !this._botCommandsService.IsMatchedWithCommand(request, template);
-                        IBotCommand command;
-                        if (isToContinue)
+                        var customCommand = await this._commandsContainer.GetCommand(request, commandInParameterType, contexts.Server.Id);
+                        if (customCommand != null || this._botCommandsService.IsMatchedWithCommand(request, template))
```

Comment:
lepiej robić krótsze ciała ifów
czyli jak wiesz że nie przejdzie dalej, to rób return zamiast dodawania wgłębienia 
bo tak jest czytelniej, bo nie trzeba sprawdzać czy przypadkiem na dole nie ma jakiegoś else 

jak dobrze pamiętam to visual studio ma wbudowaną opcje odwracania ifów w taki sposób

Reply:
czyli odwrócić warunek ifa i w klamrach dać return/continue?

Reply:
tak